src/olympia/scanners/models.py

```diff
@@ -306,6 +306,8 @@ class ScannerQueryRule(AbstractScannerRule):
     run_on_disabled_addons = models.BooleanField(
         default=False, help_text=_('Run this rule on add-ons that have been '
                                    'force-disabled as well.'))
+    group_result_id = models.UUIDField(default=None, null=True)
```

Comment:
I feel like this either deserves a comment or should be named `celery_group_result_id` to make it clearer what this is about.

Reply:
b45b8c39af10c3b07d54b85ac82cccbfb249b534